src/bundle/KiotaRequestAdapter.cs

```diff
@@ -0,0 +1,47 @@
+﻿// ------------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.  See License in the project root for license information.
+// ------------------------------------------------------------------------------
+
+using System.Net.Http;
+using Microsoft.Kiota.Abstractions;
+using Microsoft.Kiota.Abstractions.Authentication;
+using Microsoft.Kiota.Abstractions.Serialization;
+using Microsoft.Kiota.Http.HttpClientLibrary;
+using Microsoft.Kiota.Serialization.Form;
+using Microsoft.Kiota.Serialization.Json;
+using Microsoft.Kiota.Serialization.Multipart;
+using Microsoft.Kiota.Serialization.Text;
+
+namespace Microsoft.Kiota
+{
+    /// <summary>
+    /// The <see cref="IRequestAdapter"/> implementation that derived from <see cref="HttpClientRequestAdapter"/> with registrations configured.
+    /// </summary>
+    public class KiotaRequestAdapter : HttpClientRequestAdapter
```

Comment:
[Open for feedback]
Naming. Is `KiotaRequestAdapter` okat for the derived instance? Or `DefaultRequestAdapter`? cc @sebastienlevert cc @baywet

Reply:
Yeah default was the only other idea I could come up with as well. I don't have a strong opinion here.

Reply:
Looping in @maisarissi and @sebastienlevert before resolving this one.

Reply:
I think I like more DefaultRequestAdapter, but I would be ok with both.

Reply:
DefaultRequestAdapter it is...